examples/fluids/navierstokes.c

```diff
@@ -763,7 +763,8 @@ int main(int argc, char **argv) {
                             "Scale coefficient for tau (nondimensional)",
                             NULL, CtauS, &CtauS, NULL); CHKERRQ(ierr);
   if (stab == STAB_NONE && CtauS != 0) {
-    ierr = PetscPrintf(comm, "Warning! Use -CtauS only with -stab su or -stab supg\n");
+    ierr = PetscPrintf(comm,
+                       "Warning! Use -CtauS only with -stab su or -stab supg\n");
```

Comment:
@valeriabarra , I guess we need to add space between the inputs in line `582`, right?

Reply:
Yup, thanks. I just did.